src/edtr-io/components/save-modal.tsx

```diff
@@ -226,17 +231,10 @@ export function SaveModal({
           }}
         />{' '}
         <span
-          className="license-wrapper"
+          className="license-wrapper [&_a]:!text-brand hover:[&_a]:underline"
```

Comment:
this syntax hurts a bit to be honest :D 
But if it works…

Reply:
it works